hack/pre-release.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -euo pipefail
+# this script updates the WMCO and OCP versions in the Makefile, base CSV, bundle and Dockerfiles.
+
+# usage:
+# on a release-4.x branch, run ./pre-release (x, y or z)
+# to update a z stream, use ./pre-release.sh z
+# for other streams, use the corresponding x y or z argument.
```

Comment:
you can move the usage() func to the top of the file and get remove this comment block so we don't have to keep 2 places updated